script.js

```diff
@@ -8,6 +8,22 @@ function GenerateHuman (name, surname, age, phoneNumber, country, id) {
     this.id = id
 }
 
-const peter = new GenerateHuman('Piotr', 'Ochodek', 35, '500532607', 'Poland', '#1')
+const peter = new generateHuman('Piotr', 'Ochodek', 35, '500532607', 'Poland', '#1')
 
-console.log(peter)
\ No newline at end of file
+console.log(peter)
+
+class GenerateHuman {
+    constructor(name, surname, age, phoneNumber, country, id) {
+    this.name = name,
+    this.surname = surname,
+    this.email = `${name}${surname}@test.pl`
+    this.age = age,
+    this.phoneNumber = phoneNumber,
+    this.country = country,
+    this.id = id
+    }
+}
+
+const monica = new GenerateHuman('Monica', 'Jones', 32, '123123123', 'Poland', '#2')
+
```

Comment:
pls add two methods, one that change user phone number, another that store old number in history, so you need history field, and after you change old number to new, you add old number to history